syntax = "proto3";

message PyroCommand {
  int64 timestamp_us = 1; // Timestamp in microseconds
  uint32 channel_id = 2; // Which pyro channel to activate (e.g., 0 for main, 1 for drogue)
  bool activate = 3; // True to activate, false to deactivate (though typically only activate is used)
}